internal/network/dual_stack_validations.go

```diff
@@ -59,3 +60,35 @@ func VerifyClusterNetworksDualStack(networks []*models.ClusterNetwork, isDualSta
 
 	return nil
 }
+
+func CheckIfDualStack(c *common.Cluster) bool {
```

Comment:
Could you document this function?

The function is called `CheckIfDualStack` but the variable holding the boolean is called `reqDualStack`. This makes it unclear on what the function should do. To this, we can add that the variable holding the returned value from this function is called `alreadyDualStack`. 

Are we checking if it is dual-stack? Or whether it should be dual stack? Or only if it could be dual stack?